### DISCUSSION TITLE
Release 22.05

### DISCUSSION THREAD

**Discussion by svpcom** [svpcom/wfb-ng#223] [General]
Changes:
-------------
1. Fixed bug with mavlink rx error report
2. CLI refactoring - now works even for very small terminal sizes
3. Removed python2 support - minimal python version now is 3.6
4. Log messages for lost packets now aggregated - no more log flooding.
5. Location of `/etc/wifibroadcast.cfg` can be customized via `WIFIBROADCAST_CFG` environment variable
6. mavlink_sys_id/comp_id for injected `RADIO_STATUS` packets can be customized via config file.

Due to (6) you need to use [osd](https://github.com/svpcom/wifibroadcast_osd) >= 22.05

<hr /><em>This discussion was created from the release <a href='https://github.com/svpcom/wifibroadcast/releases/tag/wifibroadcast-22.05'>Release 22.05</a>.</em>

**Comment by glory86** [comment]
i'v downloaded the prebuilt image of this release
does it have 5.2.20 driver or which one?

regards

**Reply by svpcom** [reply]
images 22.05-rc1 and later have 5.2.20 driver. 22.05-beta* have 5.6.4.2 driver. You can check driver version via `modinfo 8812au_wfb`
